test/api/deletion_finding_and_printing_test.cpp

```diff
@@ -1,18 +1,26 @@
 #include <gtest/gtest.h>
 #include <gmock/gmock.h>
 
+#include <fstream>
+#include <sstream>
+
 #include "find_deletions/deletion_finding_and_printing.hpp"
 
 TEST(group1, test_std_out)
 {
-    std::string expected{"##fileformat=VCFv4.2\n"
-                         "##source=iGenVarCaller\n"
-                         "CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO\n"
-                         "chr21	9435236	.	N	<DEL>	60	PASS	SVTYPE=DEL;SVLEN=-50;END=9435286\n"
-                         "chr21	11104574	.	N	<DEL>	60	PASS	SVTYPE=DEL;SVLEN=-248;END=11104822\n"};
+    find_and_print_deletions(DATADIR"detect_breakends_shorted.vcf", DATADIR"find_deletions_file_out_test.vcf");
+    std::ifstream f;
+    f.open(DATADIR"find_deletions_file_out_test.vcf");
+    std::stringstream buffer;
+    buffer << f.rdbuf();
```

Comment:
I got the tip from jörg to use EXPECT_RANGE_EQ when comparing a test file. With you you don't need to create a test file for the three lines, but you can probably use that here and it will be shorter?
My code from `test/cli/detect_breakends_cli_test.cpp`:
```cpp
TEST_F(cli_test, test_outfile)
{
    cli_test_result result = execute_app("detect_breakends",
                                         data("simulated.minimap2.hg19.coordsorted_cutoff.sam"),
                                         "detect_breakends_insertion_file_out.fasta");

    std::filesystem::path out_file_path = "detect_breakends_insertion_file_out.fasta";
    std::filesystem::path test_file_path = "../../data/detect_breakends_insertion_file_test.fasta";
    seqan3::sequence_file_input out_file{out_file_path};
    seqan3::sequence_file_input test_file{test_file_path};

    EXPECT_RANGE_EQ(out_file, test_file);
}
```
my idea:
```suggestion
    find_and_print_deletions(DATADIR"detect_breakends_shorted.vcf", DATADIR"find_deletions_file_out_test.vcf");

    std::filesystem::path out_file_path = "find_deletions_file_out_test.vcf";
    seqan3::sequence_file_input out_file{out_file_path};
```
But I don't know whether that is possible and whether "expected" has to be packed differently ...

Reply:
So I found out that the expect_range_eq only makes sense if we use the vcf parser of SeqAn because expect_range_eq compares ranges over records and if we just read the file with a stream, we don't have any records. 
So should we add the vcf parser or leave it as is ?

Reply:
You are absolutely right. Leave it as it is and we will do that when the vcf parser is there.
It still has to be implemented in seqan ;)